pkgs/development/libraries/libsignon-glib/default.nix

```diff
@@ -2,17 +2,17 @@
 
 stdenv.mkDerivation rec {
   pname = "libsignon-glib";
-  version = "3639a2e90447e4640a03a44972560afe8f61aa48";
+  version = "2.1";
 
-  name = "${pname}-2018-10-24";
+  name = "${pname}-${version}";
 
   outputs = [ "out" "dev" "devdoc" "py" ];
 
   src = fetchgit {
     url = "https://gitlab.com/accounts-sso/${pname}";
```

Comment:
You could use `fetchFromGitLab` here

Reply:
They use a submodule for the Signon DBus specification that's shared across their projects. 
`fetchFromGitLab` won't get the submodule.

Reply:
Ah yes, I forgot it does not understand `fetchSubmodules = true` like `fetchFromGitHub`

Reply:
Actually just realized the `rev` should be ` rev = "refs/tags/${version}";`